Assets/Scripts/GameStructure/Act.cs

```diff
@@ -40,27 +43,23 @@ private void LoadActData()
         SaveSystem saveSystem = SaveSystem.Instance;
         SaveSystem.UserData userData = saveSystem.GetUserData();
 
-        //We've surpassed this act, unlock all chapters.
-        if (userData.currentAct > actNumber)
-        {
-            foreach (ChapterStruct chapter in chapters)
-            {
-                chapter.isComplete = true;
-            }
-        }
-        // We're current at this act
-        else if (userData.currentAct == actNumber)
+        starsEarnedThisAct = 0.0f;
+        foreach (SaveSystem.ChapterSaveData saveData in userData.chapterSaveData)
```

Comment:
Do you find this readable enough with a LINQ expression?

        // Only compare against chapters in this act
        foreach (var saveData in userData.chapterSaveData.Where(saveData => saveData.actNumber == actNumber))

Reply:
I ama  tidy kiwi so I prefer them but the if checks are perfectly fine too

Reply:
Didn't know these existed, I like. Prefer not to use var though for readability. 👍 🙇‍♀️